### PR TITLE
[POM]: Normalised POM and removed loading of dependencies from Github…

### PR DESCRIPTION
**Pull request by meletev**
… Packages.

### CODE REVIEW

**Comment by bertrand-lorentz** [comment]
In order for the dependencies to work after this, they need to be updated to versions that are available on Maven Central:
efx-toolkit-java to 0.1.1:
`<efx.toolkit.version>0.1.1</efx.toolkit.version>`

eforms-sdk to 0.6.2:
```
<artifactId>eforms-sdk</artifactId>
<version>0.6.2</version>
```

**Review by bertrand-lorentz — APPROVED** [review]
New commit fixes the dependencies versions, so OK for me.

**Review by bertrand-lorentz — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -72,17 +97,16 @@
...
-  </repositories>
-
   <build>
     <pluginManagement>
       <plugins>
+        <plugin>
+          <artifactId>maven-install-plugin</artifactId>
+          <version>2.5.2</version>
+          <configuration>
+            <createChecksum>true</createChecksum>
```
Comment (rouschr) [inline pom.xml:107]: What kind of checksum will it do by default? SHA-1?

**Review by rouschr — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (meletev) [reply pom.xml:107]: The default is MD5 and SHA-1.

**Review by rousso — APPROVED** [review]

**Pull request merged**